### PR TITLE
Feature/ignore selector

### PR DESCRIPTION
**Pull request by LoeiFy**
on antd table with `expandedRowRender ` option [link](https://ant.design/components/table/#components-table-demo-expand)

when open the `expand` content, drag list will cause errors

screen shot 2019-01-13 at 20 02 27

the error cause by `getDomIndex` method, it return error `DOM` index

add option `ignoreSelector` for fixing this

### CODE REVIEW

File: examples/table.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      },
+    ];
+    const dragProps = {
+      onDragEnd: (fromIndex, toIndex) => {
+        const { dataSource } = this.state;
+        const item = dataSource.splice(fromIndex, 1)[0];
+        dataSource.splice(toIndex, 0, item);
+        this.setState({ dataSource });
+      },
+      handleSelector: 'a',
```
Comment (raisezhang) [inline examples/table.js]: The dragged line will select the expanded row， add prop   ```nodeSelector: ".ant-table-row",```

File: src/ReactDragListView.jsx [hunk 2]
```diff
@@ -24,6 +25,7 @@ class ReactDragListView extends Component {
 
   static defaultProps = {
     nodeSelector: DEFAULT_NODE_SELECTOR,
+    ignoreSelector: undefined,
```
Comment (raisezhang) [inline src/ReactDragListView.jsx]: Recommended to be consistent with the default type of handleSelector

File: package.json [hunk 3]
```diff
@@ -56,6 +56,7 @@
     "prop-types": "^15.5.8"
   },
   "devDependencies": {
+    "antd": "^3.12.2",
```
Comment (raisezhang) [inline package.json]: Put Antd's Demo on CodePen ?   you can fork this link [react-drag-listview  for ant-design's table columns](https://codepen.io/raisezhang/pen/MoMoyz)  and change a suitable title!

File: examples/table.js (hunk 1 above)
Comment (LoeiFy) [reply examples/table.js]: add prop `nodeSelector: '.ant-table-row'` still throw errors

Comment (raisezhang) [reply examples/table.js]: I have tested it and found no problems. Can you provide a test link for trouble?

File: package.json (hunk 3 above)
Comment (LoeiFy) [reply package.json]: added

File: src/ReactDragListView.jsx (hunk 2 above)
Comment (LoeiFy) [reply src/ReactDragListView.jsx]: fixed

File: examples/table.js (hunk 1 above)
Comment (LoeiFy) [reply examples/table.js]: https://codepen.io/anon/pen/BvvbWv

move the last line, and you can see the error

Comment (raisezhang) [reply examples/table.js]: I recommend adding attributes based on the PR you submitted.     It has not been released yet！

**Comment by raisezhang** [comment]
I modified the Example link just to make it easy to upgrade the new version.   

https://codepen.io/raisezhang/pen/OrrGJL      

Thanks~

**Pull request merged**